packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -121,30 +124,49 @@ export default class Editor {
         }
     }
 
+    /**
+     * Whether the editor is disposed
+     */
     public isDisposed(): boolean {
         return this.disposed;
     }
 
+    /**
+     * Get the selection range
+     */
     public getSelectionRange(): Range {
         // When we have the focus, we should try to do a live pull on the selection.
         // Otherwise, return whatever we have in cache.
         let selectionRange = this.hasFocus() ? tryGetSelectionRange(this.contentDiv) : null;
         return selectionRange || this.cachedSelectionRange;
     }
 
+    /**
+     * Get the document at contentDiv
+     */
     public getDocument(): Document {
         return this.contentDiv.ownerDocument;
     }
 
+    /**
+     * Get current selection
+     */
     public getSelection(): Selection {
         return getSelection(this.getDocument());
     }
 
+    /**
+     * Whether editor has focus
+     */
     public hasFocus(): boolean {
         let activeElement = this.getDocument().activeElement;
         return activeElement && (this.contentDiv == activeElement || this.contains(activeElement));
     }
 
+    /**
+     * Put focus on editor
+     * @param resetCursor Whether to reset cursor, if resetCursor is requested, set selection to beginning of content
```

Comment:
Please add comment that the default value is false.

Reply:
Added